### ISSUE TITLE
Chrome debugging port

### ISSUE THREAD

**Issue by usaccounts** [Kiyozz/electron-esbuild#11]
Options should be added for remote debugging in chrome - hard coded example below.

```+++ b/packages/electron-esbuild/src/commands/dev.ts
@@ -46,7 +46,7 @@ class _ApplicationStarter {
     _logger.log('Start application')
     this._electronProcess = spawn(
       path.resolve(path.resolve(`node_modules/.bin/${_electronBin}`)),
-      ['dist/main/main.js'],
+      ['dist/main/main.js', '--remote-debugging-port=9222'],
     )

**Comment by Kiyozz** [comment]
Good issue! This will come in next release!

**Comment by Kiyozz** [comment]
Fixed in @electron-esbuild@1.5.0
